mission-7/queries.sql

```diff
@@ -0,0 +1,126 @@
+/*
+  다음 경우들에 대해 총 14개의 SQL 쿼리를 작성해 주세요.
+  예시로 값이 필요한 경우 적당한 값으로 채워넣어서 작성하면 됩니다. 
+*/
+
+/*
+  1. 내 정보 업데이트 하기
+  - 닉네임을 "test"로 업데이트
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+-- INSERT INTO "User" (email, nickname, password) VALUES ('test@test.com', 'user1', '12345678a');
+UPDATE "User" SET nickname = 'test' WHERE id = 1;
+-- SELECT * from "User";
+
+/*
+  2. 내가 생성한 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/
+-- INSERT INTO "Product" (name, description, price, tags, "userId") VALUES ('test 상품', '테스트입니다.', 10000, ARRAY['test1', 'test2'], 2);
+SELECT * FROM "Product" WHERE "userId" = 1 LIMIT 10 OFFSET 20;
+-- SELECT * from "Product";
+
+/*
+  3. 내가 생성한 상품의 총 개수
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+SELECT COUNT(*) AS "총 개수" FROM "Product" WHERE "userId" = 1; 
+
+/*
+  4. 내가 좋아요 누른 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/
+SELECT p.* 
+FROM "Product" as p
+JOIN "ProductLike" as pl
+on p.id = pl."productId"
+WHERE pl."userId" = 1
+ORDER BY pl."createdAt" DESC
+LIMIT 10
+OFFSET 20;
+
+/*
+  5. 내가 좋아요 누른 상품의 총 개수
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+SELECT COUNT(p.*) 
+FROM "Product" as p
+JOIN "ProductLike" as pl
+on p.id = pl."productId"
+WHERE pl."userId" = 1;
```

Comment:
지금 설계하셨던 스키마에서는, ProductLike만 조회해도 총 개수 자체는 가져올 수 있을 것 같아요.
```suggestion
SELECT COUNT(*) AS "총 개수"
FROM "ProductLike"
WHERE "userId" = 1;
```